pathspider/cmd/upload.py

```diff
@@ -0,0 +1,284 @@
+
+import pycurl
+import argparse
+import logging
+import json
+import bz2
+import dateutil.parser
+import os
+
+from io import BytesIO
+
+BASELINK = "https://v3.pto.mami-project.eu/raw/"
```

Comment:
This should be overridable, but is perhaps a sensible default for now. Please add an argument to allow overriding it.